### PR TITLE
Add location to mood event

### LINKED ISSUE #37: [US 06.01.01] Attach location to mood event
**Description:** As a participant, I want to optionally attach my current location to a mood event.

**Rationale:** So that I can associate a location with a specific mood, to better understand why I am feeling that mood.

### PR DESCRIPTION
**Pull request by WJXHenry**
Closes #37 

**Summary**

Enables the attachment of the user's current location or a specified place obtained using Google's [Place Autocomplete](https://developers.google.com/places/android-sdk/autocomplete) to a mood event.

**Notes:**
- **IMPORTANT:** The API has changed since Places Autocomplete needs a project with billing enabled. Please download the new [`secrets.xml`](https://drive.google.com/file/d/1J2CXw8IjOemjULkFQFjIbXWz8BFVt9Lh/) file with the updated API key.
- The mood event viewer fragment is not yet updated to view the location of the mood event. This is partly because of the location being stored as `GeoPoint`s. I'm suggesting that we add another field to the mood event like `locationDescription` so that the location can be associated with the place.

### CODE REVIEW

File: Moodlet/app/src/main/java/com/cmput3owo1/moodlet/activities/MoodEditorActivity.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 import android.content.Intent;
 import android.os.Bundle;
+import android.widget.Toast;
```
Comment (darrentran) [inline Moodlet/app/src/main/java/com/cmput3owo1/moodlet/activities/MoodEditorActivity.java]: Is this import used?

**Comment by darrentran** [comment]
Has the `secrets.xml` linked in the README also been updated?

File: Moodlet/app/src/main/java/com/cmput3owo1/moodlet/activities/MoodEditorActivity.java (hunk 1 above)
Comment (WJXHenry) [reply Moodlet/app/src/main/java/com/cmput3owo1/moodlet/activities/MoodEditorActivity.java]: Nope. I forgot to get rid of that after testing.

**Review by WJXHenry — COMMENTED** [review]
@darrentran It's the same link. I just updated the file on Google Docs.

**Review by darrentran — APPROVED** [review]

**Pull request merged**